src/main/java/dev/blachut/svelte/lang/SvelteJSReferenceExpressionResolver.kt

```diff
@@ -0,0 +1,98 @@
+// Copyright 2000-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+package dev.blachut.svelte.lang
+
+import com.intellij.lang.javascript.psi.JSEmbeddedContent
+import com.intellij.lang.javascript.psi.impl.JSReferenceExpressionImpl
+import com.intellij.lang.javascript.psi.resolve.JSReferenceExpressionResolver
+import com.intellij.lang.javascript.psi.resolve.JSResolveResult
+import com.intellij.lang.javascript.psi.resolve.ResolveResultSink
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import com.intellij.psi.ResolveResult
+import com.intellij.psi.util.PsiTreeUtil
+import com.intellij.psi.xml.XmlTag
+import com.intellij.xml.util.HtmlUtil
+import dev.blachut.svelte.lang.psi.*
+
+class SvelteJSReferenceExpressionResolver(referenceExpression: JSReferenceExpressionImpl,
+                                          ignorePerformanceLimits: Boolean) :
+    JSReferenceExpressionResolver(referenceExpression, ignorePerformanceLimits) {
+    override fun resolve(expression: JSReferenceExpressionImpl, incompleteCode: Boolean): Array<ResolveResult> {
+        return if (myUnqualifiedOrLocalResolve) {
+            resolveInComponent(expression, incompleteCode) ?: arrayOf()
```

Comment:
Could fallback to `super.resolve(expression, incompleteCode)` to resolve expressions like `Math.pow(1, 2)`

Reply:
I'll probably do it like you've said for now. I'm tinkering with something better so there won't be false-positives but with no luck 😢 And better to release at least something something

Reply:
Well, this looks awesome already! Anyway, I can help with debugging if something is acting really weird.

Reply:
I'm not taking about bugs. IDE sometimes picks up references from other files and links them. Now I've removed if from resolve and applied your suggestion and it magically started working better so I don't understand this stuff anymore 😅